server/src/core/auth/services/auth.service.ts

```diff
@@ -101,4 +136,16 @@ export class AuthService {
 
     return { exists: !!user };
   }
+
+  async checkWorkspaceInviteHashIsValid(
```

Comment:
@charlesBochet  This function and the resolver of this function are related to workspace only, they should be placed in the workspace module directly instead of auth